Docs for SVM in new interfaces (#1323)
oneapi-src/oneDAL: docs/source/onedal/kernel-functions/rbf-kernel.rst

.. |c_op| replace::     `compute(...) <rbf_kernel_c_api_>`_

=============== =============== ======== =========== =================
 **Operation**  **Computational methods**   **Programming Interface**
--------------- ------------------------ -----------------------------
   |c_math|      |c_dense|       |c_op|   |c_input|   |c_result|
=============== =============== ======== =========== =================

------------------------
Mathematical formulation
------------------------

.. _rbf_kernel_c_math:

Computing
---------

Given a set :math:`X` of :math:`n` feature vectors :math:`x_1 = (x_{11}, \ldots, x_{1p}), \ldots, x_n = (x_{n1}, \ldots, x_{np})`
of dimension :math:`p` and a set :math:`Y` of :math:`m`
feature vectors :math:`y_1 = (y_{11}, \ldots, y_{1p}), \ldots, y_m = (y_{m1}, \ldots, x_{mp})`,
the problem is to compute the RBF kernel function :math:`K(x_i,, y_i)` for any pair of input vectors:

.. math::
   K\left({x}_{i},{y}_{j}\right)=exp\left(-\frac{{\left(\|{x}_{i}-{y}_{j}\|\right)}^{2}}{2{\sigma }^{2}}\right)

.. _rbf_kernel_c_math_dense:

---------------------
Programming Interface
---------------------

> All types and functions in this section shall be declared in the
> ``oneapi::dal::rbf_kernel`` namespace and be available via inclusion of the
> ``oneapi/dal/algo/rbf_kernel.hpp`` header file.

Reviewer: ```suggestion
All types and functions in this section are declared in the
``oneapi::dal::rbf_kernel`` namespace and are available via inclusion of the
``oneapi/dal/algo/rbf_kernel.hpp`` header file.
```
